src/AltinnCore/Common/Services/Implementation/InstanceAppSI.cs

```diff
@@ -78,26 +78,32 @@ public async Task<Instance> InstantiateInstance(StartServiceModel startServiceMo
                 serviceModel,
                 instanceId,
                 serviceImplementation.GetServiceModelType(),
-                applicationOwnerId,
-                applicationId,
+                org,
+                appId,
```

Comment:
as far as I can see the InsertData method is expecting appName and not appId in this field. Could you double check?

Reply:
I believe it is appName

Reply:
shouldn't we the insert the appName and not appId?

Reply:
Yup. has fixed.